ebos/eclgenericoutputblackoilmodule.cc

```diff
@@ -1133,11 +1123,25 @@ outputRegionFluidInPlace_(std::unordered_map<Inplace::Phase, Scalar> oip,
         return;
 
     // don't output FIPNUM report if the region has no porv.
-    if (cip[Inplace::Phase::PoreVolume] == 0)
+    if (! (cip[Inplace::Phase::PoreVolume] > Scalar{0}))
```

Comment:
I would *really* prefer the microsteps to go away from the `Scalar & ScalarBuffer`rather than the opposite.